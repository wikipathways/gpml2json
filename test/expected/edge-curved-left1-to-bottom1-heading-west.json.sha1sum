1911178e7a9a3f928fc52069e8e5e4d805baf68b ?-
