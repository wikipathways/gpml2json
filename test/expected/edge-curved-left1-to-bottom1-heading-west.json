{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["e8d93","a9dcb","a90f1"],"drawAs":"rect","gpmlElementName":"Pathway","height":139.24230153957924,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":244.43112626346007,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/dedddb79c5433ff7f0fd4a92ac7584dc108fc6476937da0e50740025ced22ee6"},"entityMap":{"a9dcb":{"id":"a9dcb","textContent":"0019","wpType":"GeneProduct","x":19.933429242389792,"y":45.42017286709599,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32999,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"a90f1":{"id":"a90f1","textContent":"GeneProduct","wpType":"GeneProduct","x":134.43112626346007,"y":89.24230153957924,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":33000,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"e8d93":{"id":"e8d93","drawAs":"CurvedLine","borderWidth":1,"zIndex":32998,"points":[{"isAttachedTo":"a90f1","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"x":134.43112626346007,"y":99.24230153957924,"orientation":[-1,0]},{"isAttachedTo":"a9dcb","attachmentDisplay":{"position":[0.5,1],"offset":[0,0]},"x":59.93342924238979,"y":65.420172867096,"orientation":[0,-1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["a90f1","a9dcb"]}}}
